config/locales/blacklight-oembed.<path>.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This file name seems suspicious...unless there is something I'm missing.

Reply:
![](https://i.imgflip.com/40i7n0.jpg)

Reply:
Let me see if I can figure it out.

Reply:
https://github.com/sul-dlss/blacklight-oembed/pull/13

Reply:
just good ol' human error. typo in the Transifex-GitHub config 😭